src/Core/src/Platform/Windows/MauiToolbar.xaml.cs

```diff
@@ -95,6 +96,18 @@ internal void SetBarTextColor(WBrush? brush)
 			UpdateMenuBarForeground();
 		}
 
+		internal void SetBarBackground(WBrush? brush)
+		{
+			if (brush is not null)
```

Comment:
What happens if you set the background to null, will it keep the old background?

Reply:
@mattleibow, yes, it will retain the old background color in this context. So, it seems that a null check is not necessary here. I have removed it and set the actual value. Please let me know if you have any concerns.